docs/en/Collaboration.md

```diff
@@ -0,0 +1,66 @@
+# Shared Workspaces with Kubeflow
```

Comment:
Can we add an overview heading at the top here that adds context to different forms of collaboration?  As I read the sections below they felt disconnected - everything made sense in isolation, but it wasn't clear why they're on the same page in the docs.  An overview might help solve that.  Perhaps something saying how collaboration within the platform takes different forms, like sharing resources (workspace), code (workspace/git), and data (...), etc.